CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Change Log
 
 2.0
 ---
+* Added support for BrowserStack
```

Comment:
Nit, which I should have spotted before; The first and third lines are past tense while the second is present tense. I try to keep all my commit messages in the present tense, and it looks like other change log comments are in the present tense.